rwlib/source/fonts/FontMapGta3.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef _RWLIB_FONTS_FONTMAPGTA3_HPP_
```

Comment:
I think this file is more at home inside engine, as it contains a lot of hard-coded game specific stuff.

Reply:
After discussion on irc, it can stay in rwlib (for now).